jujugui/static/gui/src/app/components/svg-icon/svg-icon.js

```diff
@@ -86,7 +86,8 @@ YUI.add('svg-icon', function() {
         <svg className={this._generateClasses()}
           viewBox={this._generateViewbox()}
           style={this._generateStyles()}>
-          <use xlinkHref={'#' + this.props.name} />
+          <use xlinkHref={'#' + this.props.name}
+            id={'bob-' + this.props.name}/>
```

Comment:
Oops, this was left over from me trying to fix the svg colours. This change can be removed.